plugins/modules/discovery_intent.py

```diff
@@ -70,9 +70,35 @@
       http_read_credential:
         description: HTTP read credentials for hosting a device
         type: dict
+        suboptions:
+            password:
+                description: HTTP(S) password and is mandatory for using HTTP credentials.
```

Comment:
Can it be bit more detail?
            description: The password for HTTP(S) authentication. Mandatory for utilizing HTTP credentials.

Reply:
Addressed